import_clusters.py

```diff
@@ -0,0 +1,153 @@
+#! /usr/bin/env python3
```

Comment:
We should probably reconcile this with the existing scripts running from cron, and move them to a more-general user than dmick or yaarit.  (crash reports are separate and don't relate, but I include them for completeness).  It could
be that versions_all_clusters.py is obsolete now.

54 * * * * /home/dmick/src/proc_reports.py
# 16 weeks history, 7 days step
16 * * * * /home/dmick/src/versions_all_clusters.py --interval 16 --step 7
30 04 * * * /home/dmick/bin/gen_crash_report telemetry@ceph.io

Reply:
Yes, how about a 'telemetry' user?
I monitor daily the output of the import scripts, so we need to make sure it's also on crontab:

```
MAILTO=yaarit@redhat.com

0 12 * * * /home/yaarit/telemetry-deploy/import_clusters.py
20 12 * * * /home/yaarit/telemetry-deploy/import_devices.py
```

Reply:
Sounds good.  We can put the scripts into the telemetry user's homedir and set up an alias for mail output that initially includes at least you.  Once the import_ scripts are in a git directory somewhere (probably in ceph/ceph-telemetry.git) we can talk to @djgalloway  about Ansible automation to set up the server in case of catastrophe, too.

Reply:
Yes, sounds good :-)